GitOut/App.xaml.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```

Comment:
odd diff 👀

Reply:
It removes the BOM character. We should probably add this preference to the editorconfig file, I don't see why we would need it in any file